### PR TITLE
Release 1.7.1

### PR DESCRIPTION
**Pull request by IvanSanchez**
Followed the release instructions to release 1.7.0, but the `uglifyjs` script was b0rked (there was an upgrade from v2 to v3, and the CLI parameter format changed). Result was that the 1.7.0 uglified file at NPM was showing a header referencing a 1.3.4 working branch instead of 1.7.0 :-(

Fixed that, rebuilt, followed the rest of the release procedure. Files for 1.7.1 are already up at the NPM repo. This PR updates all the website (including docs, announcement, links)

cc @mourner @johnd0e @cherniavskii

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,63 @@ Leaflet Changelog
 
 (all changes without author notice are by [@mourner](https://github.com/mourner))
 
-## 1.6.0
+## 1.7.0 (2020-09-03)
```
Comment (mourner) [inline CHANGELOG.md:6]:
```suggestion
## 1.7.1 (2020-09-03)
```

File: docs/_posts/2020-09-03-leaflet-1.7.1.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Announcing Leaflet 1.7.1
```
Comment (mourner) [inline docs/_posts/2020-09-03-leaflet-1.7.1.md:3]: The "Announcing" in every title gets a little weird... Maybe let's just use `Leaflet 1.7 released` as the title? Also maybe worth adding a note about us having a hiccup with `1.7.0` and hence `1.7.1`.

**Review by mourner — APPROVED** [review]
Thanks so much for doing the release @IvanSanchez! Long time overdue 🙏

**Comment by ebrensi** [comment]
ok good.  I was wondering what this 1.7.1 update was.  Thanks!

**Comment by cherniavskii** [comment]
Awesome!

File: docs/_posts/2020-09-03-leaflet-1.7.1.md (hunk 2 above)
Comment (fishergisdldt) [reply docs/_posts/2020-09-03-leaflet-1.7.1.md]: I have read all the Leaflet code。and it is well designed.then i have a question：can we add webgl to the leaflet,and 3D？

**Pull request merged**